salt/metalk8s/salt/master/accept_keys.sls

```diff
@@ -0,0 +1,6 @@
+Accept keys:
+  module.run:
+    - cri.execute:
+      - salt-master
```

Comment:
Is this the `name`? If so, can it be a named argument?

Reply:
Yes, that's the name of the targeted container. => #822